DependencyInjection/SonataUserExtension.php

```diff
@@ -52,7 +52,11 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('google_authenticator.xml');
         $loader->load('twig.xml');
 
-        if ('orm' === $config['manager_type'] && isset($bundles['FOSRestBundle']) && isset($bundles['NelmioApiDocBundle'])) {
+        if ('orm' === $config['manager_type'] && isset(
+            $bundles['FOSRestBundle'],
+            $bundles['NelmioApiDocBundle'], 
```

Comment:
StyleCI doesn't like the trailing space, and maybe you should configure your IDE to remove them on save. Otherwise it looks good :+1: